spidermon/contrib/actions/telegram/__init__.py

```diff
@@ -25,7 +25,9 @@ def send_message(self, message, recipient):
         )
         r = requests.get(api_url).json()
         if r.get("ok") is False:
-            logger.error("Failed to send message. Telegram api error: %s", r.text)
+            logger.error(
+                "Failed to send message. Telegram api error: %s", r.description
```

Comment:
Could you add a test for this change?

I believe this will simply cause the error message to change to `AttributeError: 'dict' object has no attribute 'description'`. The fix is probably to change `r.description` to `r['description']`.

Beyond that, I’m not aware of whether `text` or `description` are the right choice. Looking at the original code, I suspect the intention was to log the whole response (and the error was not to take into account that `.json()` was used). In that case, I think the best option would be to change `r.text` to `json.dumps(r)`.

Reply:
**Fix for action log**:

@Gallaecio you are right, the following would be the error message for [the recommended change](https://github.com/scrapinghub/spidermon/pull/241/commits/a79c82b09fecaf31a343ca508ba9167342037eaa):
```
ERROR: SendTelegramMessageSpiderFinished
----------------------------------------------------------------------
Traceback (most recent call last):
  File "c:\users\samuel\anaconda3\lib\site-packages\spidermon\core\actions.py", line 37, in run
    self.run_action()
  File "c:\users\xxxx\anaconda3\lib\site-packages\spidermon\contrib\actions\telegram\__init__.py", line 95, in run_action
    self.manager.send_message(to=self.recipients, text=self.get_message())
  File "c:\users\xxxx\anaconda3\lib\site-packages\spidermon\contrib\actions\telegram\__init__.py", line 49, in send_message
    self._client.send_message(text, recipient)
  File "c:\users\xxxx\anaconda3\lib\site-packages\spidermon\contrib\actions\telegram\__init__.py", line 29, in send_message
    "Failed to send message. Telegram api error: %s", r.description
AttributeError: 'dict' object has no attribute 'description'
```

Your following [suggestion](https://github.com/scrapinghub/spidermon/pull/241#discussion_r386129261) seems to be the solution for the problem above.
>The fix is probably to change r.description to r['description'].
This works for me.

Thank you very much!

Reply:
Hummm shouldn't the build failed in this case? I'll add a test and make the change anyway. Thanks for your input.